server/src/main/java/org/elasticsearch/Build.java

```diff
@@ -249,6 +249,14 @@ public String qualifiedVersion() {
         return version;
     }
 
+    /**
+     * Get the build version without any suffices like "-SNAPSHOT" or "-rc1".
+     * @return the build version with no suffix
+     */
+    public String unqualifiedVersion() {
```

Comment:
I'm curious, why do we need `unqualifiedVersion` if this is going to be an arbitrary string anyway (e.g. in Serverless?)

Reply:
It needs to be parsed as a version by pre-8.10 nodes. So nodes that might end up in a mixed version cluster with pre-8.10 nodes need to set it to a string that `Version.fromString` in those old versions of the code will be able to parse.

So effectively:

- Serverless nodes can set it to anything
- 9.x nodes can set it to anything
- 8.x nodes need to set it to 8.x.y

Reply:
Per discussion on Slack, I'll just hardcode to 8.11.0, as that will work everywhere.